src/core/bootstrap/bootstrap.service.ts

```diff
@@ -184,6 +183,7 @@ export class BootstrapService {
 
   @Profiling.api
   async createGroupProfiles(groupName: string, usersData: any[]) {
+    const defaultEcoverse = await this.ecoverseService.getFirstEcoverseOrFail();
```

Comment:
getDefaultEcoverse might be a better name

Reply:
It was getDefaultEcoverse - but on reflection we are actually taking the first ecoverse, so I updated it. Can chat it through...